### PR TITLE
Game initialisation

### PR DESCRIPTION
**Pull request by Darius1325**
Initialisation of game classes

### CODE REVIEW

File: game/player/long_john.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 
     def __init__(self):
-        Player.__init__(self, "#000000")
\ No newline at end of file
+        Player.__init__(self, "#000000")
+        self.name = "Long John Silver"
+        self.is_free = False
+        self.personal_map = None
+        self.bluffs = 0
+        self.hints = []
```
Comment (7evy) [inline game/player/long_john.py:17]: no newline wesh

File: game/player/player.py [hunk 2]
```diff
@@ -1,16 +1,28 @@
 import string
+from position import Position
```
Comment (7evy) [inline game/player/player.py:2]:
```suggestion
from game.position import Position
```

File: game/player/player.py [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
 
 from game.map.interactive_map import InteractiveMap
 
 
 class Player:
+    
+    name : string
+    position : Position
+    color : string
```
Comment (7evy) [inline game/player/player.py:11]:
```suggestion
    name : str
    position : Position
    color : str
```

File: game/player/long_john.py [hunk 4]
```diff
@@ -1,10 +1,17 @@
 from game.player.player import Player
-
+from game.map.long_john_map import LongJohnMap
 
 class LongJohn(Player):
-
-    id = "longjohn"
```
Comment (TonyChouteau) [inline game/player/long_john.py:6]: Used to link to image - Do not remove

Comment (7evy) [reply game/player/long_john.py:6]: Don't listen to me when I talk

File: game/player/player.py [hunk 5]
```diff
@@ -1,11 +1,12 @@
 import string
-from position import Position
+from game.position import Position
 
 from game.map.interactive_map import InteractiveMap
 
 
 class Player:
     
+    id : string
```
Comment (TonyChouteau) [inline game/player/player.py:9]: str not string

**Review by 7evy — APPROVED** [review]

**Review by TonyChouteau — APPROVED** [review]

**Pull request merged**